examples/AeroSharp.Examples/Maps/MapExampleData.cs

```diff
@@ -19,4 +20,35 @@ internal static class MapExampleData
         { "key9", "value9" },
         { "key10", "value10" }
     };
-}
+
+/*
+ * A nested map example
+ * {
+ *   { "key1" : {
+*        { "key21", 999.99 },
+ *       { "key22", 999.99 }
+ *   },
+ *   { "key2" : {
+ *       { "a", 5 },
+ *       { "c", 3 }
+ *   }
+ * }
+ */
+    public static readonly Dictionary<string, object> m3 = new()
+    {
+        {
+            "key1", new Dictionary<object, object>
+            {
+                { "key21", 999.99 },
+                { "key22", 999.99 }
+            }
+        },
+        {
+            "key2", new Dictionary<object, object>
+            {
+                { "a", 5 },
+                { "c", 3 }
+            }
+        }
+    };
+}
```

Comment:
Newline needs to be at EOF to be consistent (see 
<img width="36" alt="image" src="https://github.com/wayfair-incubator/AeroSharp/assets/5827236/45cba47a-8d0b-4c45-98ab-295d9c985a56"> )

Reply:
Will want to address this everywhere.

Reply:
I probably need to adjust my Rider settings.  Definitely thought I formatted each file, but great catch and thanks for sticking to style!